### ISSUE TITLE
Error with AWS Redis

### ISSUE THREAD

**Issue by olchick** [chatwoot/charts#139]
Hello!
I encountered a similar issue with AWS Redis configured without a password, as discussed in [this issue](https://github.com/chatwoot/charts/issues/110)

When I set the REDIS_URL environment variable without a password, I encountered the following error:
`Redis::CommandError (ERR AUTH <password> called without any password configured for the default user. Are you sure your configuration is correct?`

It's very strange because, to fix the error, I need to manually remove the environment variable REDIS_PASSWORD="redis"

However, I need a way to avoid adding the REDIS_PASSWORD environment variable in the secret within the Helm chart and stop remove it manually after each update.

**Comment by kostorub** [comment]
Yes, the `redis` section is useless if you want to use Redis without a password.
I fixed it the same way by setting values at the bottom of the values.yaml file:
```
  REDIS_PASSWORD: ""
  REDIS_URL: redis://redis.local
```

**Comment by olchick** [comment]
Hi @kostorub!
Thanks for your answer! 
It seems like it fixed my issue!
